src/readers/EnvReader.php

```diff
@@ -27,29 +27,29 @@ public function readRaw($path)
         }
         $info = pathinfo($path);
         $dotenv = $this->createDotenv($info['dirname'], $info['basename']);
-        $dotenv->overload();
-
+        if (method_exists($dotenv, 'load')) {
+            $dotenv->load();
+        } else {
+            $dotenv->overload();
+        }
```

Comment:
I'm not sure about this part. Need to research.

Reply:
You are right, it should be opposite, because older versions also have method `load()`.

```php
if (method_exists($dotenv, 'overload')) {
    $dotenv->overload();
} else {
    $dotenv->load();
}
```